tests/FromCommunity/Issue3Test.php

```diff
@@ -5,20 +5,17 @@
  */
 declare(strict_types=1);
 
-
 namespace OpenAPIValidationTests\FromCommunity;
 
-
-use cebe\openapi\Reader;
 use GuzzleHttp\Psr7\ServerRequest;
 use OpenAPIValidation\PSR7\ServerRequestValidator;
 use PHPUnit\Framework\TestCase;
 use function GuzzleHttp\Psr7\stream_for;
 
-class ScayTraseTest extends TestCase
+class Issue3Test extends TestCase
```

Comment:
I'm pretty proud for having named tests, but these are spawning fast. I suggest rename them according to fixed issue

Reply:
Haha, yeah. Good point.